CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+[8.1.1] - 2023-06-08
```

Comment:
I would call this 8.2.0. usually x.x.1 means a bug patch rather than a new feature or model

Reply:
Updated.